src/test/resources/google/cloud/compute/v1small/compute.merge.proto.baseline

```diff
@@ -327,7 +327,7 @@ message DeleteAddressRequest {
   // For example, consider a situation where you make an initial request and the request times out. If you make the request again with the same request ID, the server can check if original operation with the same request ID was received, and if so, will ignore the second request. This prevents clients from accidentally creating duplicate commitments.
   //
   // The request ID must be a valid UUID with the exception that zero UUID is not supported (00000000-0000-0000-0000-000000000000).
-  optional string request_id = 37109963;
+  optional string request_id = 37109963 [(google.api.field_behavior) = REQUIRED];
```

Comment:
(totally meant to send this comment this morning but didn't press the button)

Reply:
This logic just preserves the old situation - if it was optional, then it will remain optional, if it idid not have optional, it will remain non-optional. This specific thing is just a test. I modified the test data to represent better the actual case in GCE.

Reply:
Awesome, thanks!